### DISCUSSION TITLE
Bug: Instead of updating the bar, many lines are printed in Jupyter-Notebok.

### DISCUSSION THREAD

**Discussion by astronom4ek** [tqdm/tqdm#1459] [Q&A]
When I add the word sleep(0.1) to Jupyter notebook (python 3.7 Anaconda) in the loop, instead of updating the bar, a lot of lines are printed. (At the same time, if you put from a smaller time  sleep(0.01), it work  correctly)

Also, if you add the word **reversed** instead of a score, only text is output, also producing many lines.
![изображение](https://user-images.githubusercontent.com/87126842/228358268-b981ba20-f297-485d-a378-e6154f08a52e.png)

But by running the same code from the console, the bar is updated correctly. But with the **reversed**, only the text is still output
![изображение](https://user-images.githubusercontent.com/87126842/228358731-43b7beb4-45e4-4a23-9a43-0277fbdf299a.png)

**Comment by nono-london** [comment]
have you tried
from tqdm.auto import tqdm 
or
from tqdm.auto_notebook import tqdm?

see example of 6 of this turtorial that works for me on notebooks

https://medium.com/@harshit4084/track-your-loop-using-tqdm-7-ways-progress-bars-in-python-make-things-easier-fcbbb9233f24#id_token=eyJhbGciOiJSUzI1NiIsImtpZCI6ImFjZGEzNjBmYjM2Y2QxNWZmODNhZjgzZTE3M2Y0N2ZmYzM2ZDExMWMiLCJ0eXAiOiJKV1QifQ.eyJpc3MiOiJodHRwczovL2FjY291bnRzLmdvb2dsZS5jb20iLCJuYmYiOjE2ODA4MDQzMDcsImF1ZCI6IjIxNjI5NjAzNTgzNC1rMWs2cWUwNjBzMnRwMmEyamFtNGxqZGNtczAwc3R0Zy5hcHBzLmdvb2dsZXVzZXJjb250ZW50LmNvbSIsInN1YiI6IjEwMjg0MTgyNTIxNTk0MjQzMTIxMSIsImVtYWlsIjoibXlzcWxwcm9qZWN0QGdtYWlsLmNvbSIsImVtYWlsX3ZlcmlmaWVkIjp0cnVlLCJhenAiOiIyMTYyOTYwMzU4MzQtazFrNnFlMDYwczJ0cDJhMmphbTRsamRjbXMwMHN0dGcuYXBwcy5nb29nbGV1c2VyY29udGVudC5jb20iLCJuYW1lIjoiTm9ubyBMb25kb24iLCJwaWN0dXJlIjoiaHR0cHM6Ly9saDMuZ29vZ2xldXNlcmNvbnRlbnQuY29tL2EvQUdObXl4WnFjaUxXUTN1RFBpZXhmQUNLdFVCcmlBMGVfU1RQaVptMHl2XzU9czk2LWMiLCJnaXZlbl9uYW1lIjoiTm9ubyIsImZhbWlseV9uYW1lIjoiTG9uZG9uIiwiaWF0IjoxNjgwODA0NjA3LCJleHAiOjE2ODA4MDgyMDcsImp0aSI6ImMxMjc1Yjk2NGIwNzEzZmU3NDNiYzYyNzU1MWNkOWNmOTkzZjY3MjkifQ.mmC9uOfWsCoduUikH-jL_wNkGexfmkJLh7kPLqPoWnynkOnrJncCiMmOnZnQ43OKscxiaCKvyxEjrCqT_wZvZyR_a9w3FW6nYXSErswoJhNT9U9eOa7a_Lu9lVAwv0gQruOs7-Rc5q6BMdFTEtz8rCKbj8MWuZwXBBx5WB3BxdDQ395H5aCW87MWsiBEXJUQwspoajxC_OQ2BggnNAurADKSG_rBaNOu-355BsF6m3pExOI-nwtOC9pCTlMtDB9YwQMsCYO-qyh53fp_RjABQJAQueTqYw9BDPC9tHf2nfRoQCNZ6UuUholSZ-Xeagd09ClAm-09iIMpwO9JEMprYg
